app/models/collection.rb

```diff
@@ -37,6 +37,10 @@ class Collection < ActiveRecord::Base
   scope :spammy, -> { where(["created_at > ? AND (name LIKE '%movie%' OR name "\
     "LIKE '%watch%' OR name LIKE '%putlocker%' OR name LIKE '%put-locker%' OR "\
     "name LIKE '%full MKV%') AND name NOT LIKE '%Watch List%'", 1.week.ago]) }
+  scope :non_watch_collections_of_today, ->(current_user) {Collection.joins(:users).select(non_watch).
```

Comment:
Line is too long. [102/80]<br>Use the `lambda` method for multi-line lambdas.<br>Block body expression is on the same line as the block start.